static/css/section3.css

```diff
@@ -1,3 +1,7 @@
+a:hover {
+    text-decoration: none !important;
+}
+
```

Comment:
I'm not sure we quite want this-- all anchor elements won't have text-decoration at all with this rule. I think we might want this for actual URL's, but just not for this specific instance. I think this might work, but need to try it out:

```
.section-3 a:hover {
    text-decoration: none;
}
```